projects/maskfeat_video/configs/maskfeat_mvit-small_8xb32-amp-coslr-300e_k400.py

```diff
@@ -0,0 +1,102 @@
+_base_ = 'mmselfsup::selfsup/_base_/default_runtime.py'
```

Comment:
This config file seems redundant

Reply:
modified this config for 8 gpus, inherited from the other one